examples/flatris/browserify/prepare-modules.js

```diff
@@ -2,7 +2,7 @@
 
 const prepareComponents = (modules) => {
   const components = {};
-  Object.keys(modules).forEach(componentName => {
+  Object.keys(modules).forEach((componentName) => {
```

Comment:
This is odd. In some case you have to remove the parenthesis and in others (like here) you have to add it. Does it have smth to do with the curly brackets?